rooms/views.py

```diff
@@ -148,4 +148,73 @@ def get(self, request):
 
             return JsonResponse({'rooms' : result}, status=200)
         except IndexError:
-            return JsonResponse({'result' : 'INVALID GEOCODE'}, status=400)
\ No newline at end of file
+            return JsonResponse({'result' : 'INVALID GEOCODE'}, status=400)
+
+class RoomDetailView(View):
+    # @UserTokenDeco
+    def get(self, request, room_id):
+        try:
+            room               = Room.objects.select_related('host').prefetch_related('image_set', 'host__socialflatform_set', 'roomoptioninfo_set', 'roomconvenience_set', 'roomreview_set').get(id=room_id)
+            user               = 2
+            # user              = request.user
+            wish               = True if Wish.objects.filter(user_id = user, room_id = room).exists()  else False
+            image_path         = [images.s3_url for images in room.image_set.all()]
+            host_profile_image = [image.profile_image for image in room.host.socialflatform_set.all()][0]
+            room_options       = [{'name':options.room_option.name, 'quantity':options.quantity} for options in room.roomoptioninfo_set.all()]
+            room_convenience   = [
+                                    {
+                                        'name':conveniences.convenience.name, 
+                                        'exist':(
+                                                    True if conveniences.convenience_id == conveniences.id\
+                                                    and conveniences.room_id == room.id else False\
+                                            )
+                                } for conveniences in room.roomconvenience_set.all()
+                            ]
+            reviews            = [
+                                    {
+                                        'name'         :review.user.name, 
+                                        'profile_image':[images.profile_image for images in review.user.socialflatform_set.all()][:1], 
+                                        'created_at'   :review.created_at.date(),
+                                        'content'      :review.content, 
+                                        'review_id'    :review.id
+                                } for review in room.roomreview_set.all().order_by('-created_at')[:7]
+                            ]
```

Comment:
아래와 같이 컨벤션 맞춰주세요~

추가로 for문 사용할 때, 복수 단수 명확히 선언해주세요~

```suggestion
            reviews = [
                {
                    'name'          : review.user.name, 
                    'profile_images' : [image.profile_image for image in review.user.socialflatform_set.all()][:1], 
                    'created_at'    : review.created_at.date(),
                    'content'       : review.content, 
                    'review_id'     : review.id
                } for review in room.roomreview_set.all().order_by('-created_at')[:7]
            ]
```

Reply:
수정완료했습니다 :)